### PR TITLE
adding masurca

### PR DESCRIPTION
**Pull request by erinyoung**
- [X] This comment contains a description of what is in the pull request.

Pilon wasn't working for my needs, so I found a new polisher : POLCA - which is part of the masurca suite of tools. I currently only have a test for POLCA - mostly because I need to find a nanopore fastq.gz file to download with corresponding fastq files without using SRA (OR! I need to figure out how to use SRA for things like this), so I'm hoping that the others work. 

- [X] Build your own docker image using a Dockerfile
  - [X] Directory structure should be name of the tool in lower case with special characters removed with a subdirectory of the version number (i.e. spades/3.12.0/Dockerfile)
  - [X] Includes the recommended LABELS
- [X] (Optional) Dockerfile is built with best practices and has been approved by a linter (such as https://hadolint.github.io/hadolint/)
- [X] Edit main README.md
- [X] Edit Program_Licenses.md
- [X] Create a simple container-specific README.md in the same directory as the Dockerfile (i.e. spades/3.12.0/README.md)
- [X] Write a GitHub actions workflow
  - [X] Should be located in .github/workflows/ and named test-<tool>.yml (i.e. .github/workflows/test-spades.yml)
  - [X] Any files required for building are located in the same directory as the Dockerfile (i.e. spades/3.12.0/my_spades_tests.sh)
  - [X] Have successfully run the workflow "Test <program name> image" in your forked repository

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ To learn more about the docker pull rate limits and the open source software pro
 | [fastq-scan](https://hub.docker.com/r/staphb/fastq-scan) <br/> [![docker pulls](https://img.shields.io/docker/pulls/staphb/fastq-scan.svg?style=popout)](https://hub.docker.com/r/staphb/fastq-scan) | <ul><li>0.4.3</li><li>0.4.4</li></ul> | https://github.com/rpetit3/fastq-scan |
 | [Filtlong](https://hub.docker.com/r/staphb/filtlong) <br/> [![docker pulls](https://img.shields.io/docker/pulls/staphb/filtlong.svg?style=popout)](https://hub.docker.com/r/staphb/filtlong) | <ul><li>0.2.0</li><li>0.2.1</li></ul> | https://github.com/rrwick/filtlong |
 | [Flye](https://hub.docker.com/r/staphb/flye) <br/> [![docker pulls](https://img.shields.io/docker/pulls/staphb/flye.svg?style=popout)](https://hub.docker.com/r/staphb/flye) | <ul><li>2.5</li><li>2.7</li><li>2.8</li><li>2.9</li></ul> | https://github.com/fenderglass/Flye |
-| [Freyja](https://hub.docker.com/r/staphb/freyja) <br/> [![docker pulls](https://img.shields.io/docker/pulls/staphb/freyja.svg?style=popout)](https://hub.docker.com/r/staphb/freyja) | <ul><li>1.2</li><li>1.2.1</li><li>1.3.1</li><li>1.3.2</li></ul> | https://github.com/andersen-lab/Freyja |
```
Comment (erinyoung) [inline README.md:64]: Looks like I have some issues with the README.md. I'm a version or two behind.

**Comment by kapsakcj** [comment]
Looks good to me! 

Want to also add a build-to-deploy workflow for this one?

**Review by kapsakcj — APPROVED** [review]

**Comment by erinyoung** [comment]
> Want to also add a build-to-deploy workflow for this one?

Yes! What do those look like?

**Comment by kapsakcj** [comment]
So I don't know if this was implemented perfectly (my first attempt) but my PR for the pangolin update includes this "build-to-deploy" workflow #311 .

Would be good to get feedback from @SarahNadeau and/or @k-florek 

It essentially runs the testing layer of your dockerfile (builds image; runs test layers), followed by building, tagging & pushing to dockerhub, followed by downloading the image just pushed to dockerhub w/ singularity. 

Looks like there is a way to feed in test commands to the singularity portion of the workflow, but I haven't figured it out yet.

You could do something similar here, like so, might call it `./github/workflows/deploy-mascura-4.0.8.yml`
```
# This caller workflow tests, builds, and pushes the image to Docker Hub.
# It also tests that a Singularity container can be run from the Docker image.

name: Deploy Mascura 4.0.8 image

on: workflow_dispatch

jobs:

  build-to-test:
    uses: StaPH-B/docker-builds/.github/workflows/build-to-test.yml@master
    with:
      path_to_context: "./masurca/4.0.8/"
      dockerfile_name: "Dockerfile"
      cache: "mascura"

  build-to-deploy:
    needs: build-to-test  # this jobs needs to run serially, after testing succeeds
    uses: StaPH-B/docker-builds/.github/workflows/build-to-deploy.yml@master
    secrets:
      docker_username: ${{ secrets.DOCKER_HUB_USERNAME }}
      docker_access_token: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
    with:
      path_to_context: "./masurca/4.0.8/"
      dockerfile_name: "Dockerfile"
      cache: "mascura"
      container_name: "mascura"
      tag: |
        4.0.8
        latest
  run-singularity:
    needs: build-to-deploy  # test singularity run on newly pushed Docker image
    uses: StaPH-B/docker-builds/.github/workflows/run-singularity.yml@master
    with:
      image_name: "mascura:4.0.8"
```

Since your GH actions workflow `.github/workflows/test-masurca.yml` tested successfully, then I think the next steps would be to:
  * add the above `deploy-mascura-4.0.8.yml` workflow
  * merge the PR into `master` branch
  * manually trigger the `deploy-mascura-4.0.8.yml` workflow via GH Actions page
  * ensure docker image is tagged and pushed appropriately to dockerhub
  * I can manually sync to quay.io via `skopeo`

**Comment by erinyoung** [comment]
Alright. I added the deploy yml file, so once this passes the test, I'd like to merge this and push the image to dockerhub.

**Comment by erinyoung** [comment]
I'm merging this because it passed Curtis's review

**Pull request merged**